gensim/test/test_word2vec.py

```diff
@@ -763,6 +763,15 @@ def testLineSentenceWorksWithNormalFile(self):
                 sentences = word2vec.LineSentence(fin)
                 for words in sentences:
                     self.assertEqual(words, utils.to_unicode(orig.readline()).split())
+
+    def testLineSentencePath(self):
+        """Does LineSentencePath work with a path argument?"""
+        logging.debug(word2vec)
+        with utils.smart_open(datapath('lee_background.cor')) as orig:
+            sentences = word2vec.LineSentencePath(datapath('LineSentencePath'))
```

Comment:
Change made, put in a new tiny test corpus, and changed the test to read the files and combine them manually and compare to PathLineSentences, rather than having a single file duplicating the contents of the PathLineSentences folder in test_data